AllReadyApp/Web-App/AllReady/Features/Events/UnregisterEventHandler.cs

```diff
@@ -1,27 +1,42 @@
 ﻿using System.Threading.Tasks;
+using System.Linq;
 using AllReady.Features.Notifications;
 using AllReady.Models;
 using MediatR;
 
 namespace AllReady.Features.Event
 {
-    public class UnregisterEventHandler : AsyncRequestHandler<UnregisterEvent>
-    {
-        private readonly IAllReadyDataAccess dataAccess;
+    public class UnregisterEventHandler : AsyncRequestHandler<UnregisterEvent> {
+        private readonly AllReadyContext dataContext;
```

Comment:
I left the prefixing of private variables when I left `m_VB`.  Interesting that we differ on preference here.  The solution seems to include many of each technique.  I tried to leave the code using the style it had rather than enforce one convention or the other.